README.md

```diff
@@ -14,7 +14,6 @@ Clone this repository
 ```
 git clone https://github.com/compassinformatics/cpsi-mapview.git
 cd cpsi-mapview
-git submodule update --init --recursive
```

Comment:
Add note about cloning dependencies to the lib folder if fixes need to be added to the dependencies (see note below).